modules/ROOT/pages/tutorial-manage-an-api.adoc

```diff
@@ -68,25 +64,24 @@ traits:
 ** Time Period: 1
 ** Time Unit: Minute
 
-== To Apply the Policy
+== Apply the Policy
 
-This procedure assumes you just finished add an SLA tier as described above, or you navigated to API Manager > API Administration. In the latter case, click an API instance, and then follow this procedure to apply a policy.
+After adding the SLA tier, the policy can now be applied to the API.
 
-. Choose Policies.
-+
-. Click Apply New Policy, and in Select Policy, select Rate Limiting - SLA-based policy, and click Configure. Alternatively, select Rate Limiting > Configure Policy.
-. In Limits, set the number of requests, time period, and units of measurement.
-. Configure the policy on the `/users` resource of the API.
-* In Method & Resource Conditions, select Apply Configurations to Specific Methods & Resources.
-* In Methods, select GET and in URI template regex.
-* In URI Template Regex, enter `/users` to apply rate limiting only to the `/users` resource, or enter `.*` to apply rate limiting to every resource URI of the API.
-* Mule 4 use: Configure header propagation for token enforcement and rate limiting policies by checking the Expose Headers checkbox.
-* Click Apply.
+. Select *Policies*.
+. Click *Apply New Policy* and select *Rate limiting - SLA-based*.
+. Click *Configure Policy*.
+. In *Limits*, set the number of requests, time period, and units of measurement.
+. Configure the policy on the API `/users` resource:
+* In *Method & Resource Conditions*, select *Apply Configurations to Specific Methods & Resources*.
+* Under *Methods*, select GET.
+* Under *URI Template Regex*, enter `/users` to apply rate limiting only to the `/users` resource, or enter `.*` to apply rate limiting to every resource URI of the API.
+* If your environment uses Mule 4, configure header propagation for token enforcement and rate limiting policies by checking the *Expose Headers* checkbox.
```

Comment:
The expose header section is for exposing the Rate Limiting headers, it has nothing to do with token enforcement.

Reply:
I think I commented this one but missed to check the change, @chris-campbell9  in this case it should be "configure header propagation for exposing the rate limiting headers .. " and I shared with you the link where these headers are explained.